lupa/_lupa.pyx

```diff
@@ -1183,9 +1191,29 @@ cdef int py_function_result_to_lua(LuaRuntime runtime, lua_State *L, object o) e
          return len(<tuple>o)
      return py_to_lua(runtime, L, o)
 
+cdef int py_to_lua_overflow(LuaRuntime runtime, lua_State *L, object o) except -1:
+    cdef const char* cb_name = "lupa_overflow_cb"
```

Comment:
This needs to be documented and tested. In fact, I wonder if this should be a Lua-level or Python-level feature. Or both.

Reply:
The overflow only occurrs when converting a Python int/long to Lua integer, so I think it should be manipulated in Lua since the value needs to be of a Lua type.

Reply:
Ok, I'll be documenting and allowing for this callback to be set from both sides (in Lua and in Python). Also, adding tests.

Reply:
Resolved in https://github.com/scoder/lupa/pull/168/commits/f112b9b6ba48622a71c5fd50c1f34e43fdec9e46